pt_BR/docs/Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+Sphinx = "==1.7.2"
+sphinx-rtd-theme = "*"
+recommonmark = "==0.4.0"
+
+[requires]
+python_version = "3.7"
```

Comment:
Renato - do you want to update your Pipfile to be the shorter version?
See https://github.com/hyperledger/fabric/pull/1394/files
```
[packages]
Sphinx = "==1.8.5"
recommonmark = "==0.4.0"
```

Reply:
It make sense, I did the adjustments in both Pipenv files